packages/cli/commands/project/dev.js

```diff
@@ -86,18 +101,50 @@ exports.handler = async options => {
     if (shouldTargetNonSandboxAccount) {
       const {
         targetAccountId: promptNonSandboxTargetAccountId,
-      } = await selectTargetAccountPrompt(accounts, true);
+      } = await selectTargetAccountPrompt(accounts, accountConfig, true);
 
       targetAccountId = promptNonSandboxTargetAccountId;
       logger.log();
     } else {
       process.exit(EXIT_CODES.SUCCESS);
     }
   } else if (createNewSandbox) {
-    logger.log(
-      'Creating new sandboxes is not supported yet. Use "hs sandbox create" and then run this command again.'
-    );
-    process.exit(EXIT_CODES.SUCCESS);
+    try {
+      await validateSandboxUsageLimits(accountConfig, DEVELOPER_SANDBOX, env);
+    } catch (err) {
+      logErrorInstance(err);
+      process.exit(EXIT_CODES.ERROR);
+    }
+    try {
+      const { name } = await developmentSandboxNamePrompt();
+      const { result } = await buildSandbox({
+        name,
+        type: DEVELOPER_SANDBOX,
+        accountConfig,
+        env,
+      });
+
+      targetAccountId = result.sandbox.sandboxHubId;
+
+      const sandboxAccountConfig = getAccountConfig(
+        result.sandbox.sandboxHubId
+      );
+      let syncTasks = await getAvailableSyncTypes(
```

Comment:
Should this be a `const`?

Reply:
Indeed it should! leftover from moving logic into that util